src/chooser.c

```diff
@@ -189,43 +189,44 @@ static void show_player_chooser_each_frame(const struct Chooser *ch, struct Choo
 
 static void show_place_chooser_each_frame(struct ChooserPlaceStuff *plcch)
 {
-	Vec3 placecenter = { plcch->pl->xsize/2, 0, plcch->pl->zsize/2 };
+	const struct Place *pl = &plcch->places[plcch->placeidx];
+	Vec3 placecenter = { pl->xsize/2, 0, pl->zsize/2 };
 
-	float d = hypotf(plcch->pl->xsize, plcch->pl->zsize);
+	float d = hypotf(pl->xsize, pl->zsize);
 	Vec3 tocamera = vec3_mul_float((Vec3){0,0.8f,1}, 1.1f*d);
 	vec3_apply_matrix(&tocamera, mat3_rotation_xz(plcch->cam.angle));
 
+	// TODO: adjust angle earlier
```

Comment:
This was needed for similar code in map editor, where you can change spinning direction, but might not be necessary here.

```suggestion
	// TODO: see if adjusting angle earlier makes it look somehow better
```